modules/common/configmap/configmap.go

```diff
@@ -105,8 +105,12 @@ func createOrPatchConfigMap(
 		return nil
 	})
 	if err != nil {
+		h.GetRecorder().Event(obj, corev1.EventTypeWarning, "ConfigMapError", fmt.Sprintf("error create/updating configmap: %s", cm.Name))
```

Comment:
re `ConfigMapError` and other events, its probably good that we create a central reason collection which then gets used in all the operators. similar to how we do it with the conditions. Otherwise I guess if events are used in the service operators, we'll see different reasons used. I could see two ways. Either we create an events pkg and have the consts for reasons in there, or they are part of the already existing pkgs, like ConfigMap reason in configmap. but then how to handle reasons for stuff we do not yet have. create extra pkgs, or better the one events pkg. thoughts?